packages/element-library/src/audio-sticker/output.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+function AudioStickerOutput() {
+  // TODO (@AnuragVasanwala): Need to move this logic here from 'packages/output/src/page.js'.
```

Comment:
Let me know if you need any help with that.

Reply:
Yes, it would be good if you can provide some help here.

Adding anything other than `amp-story-audio-sticker` reflects into story but not Audio Sticker.

Reply:
What do you mean by "reflects into story"?

Reply:
Say, following line outputs label to the story:
```js
function AudioStickerOutput() {
  // TODO (@AnuragVasanwala): Need to move this logic here from 'packages/output/src/page.js'.
  return <div>{'AudioStickerOutput'}</div>;
}
```

while this doesn't show anything:
```js
function AudioStickerOutput() {
  // TODO (@AnuragVasanwala): Need to move this logic here from 'packages/output/src/page.js'.
  //return <div>{'AudioStickerOutput'}</div>;
  return (
    <amp-story-grid-layer template="vertical" class="grid-layer">
      <amp-story-audio-sticker size="large" sticker="tape-player" />
    </amp-story-grid-layer>
  );
}
```
Also, even markup of the `amp-story-audio-sticker` is removed from the story. It seems like it is being sanitised somehow!

Reply:
> while this doesn't renders Audio Sticker:

Note that the `*Output()` component must not include `amp-story-grid-layer`. `amp-story-grid-layer` is already added by `packages/output/src/page.js`. Otherwise it breaks AMP validation.

----

So, I just pushed some changes to "fix" the output. Turns out the sanitization happens because of a limitation in the  expected document structure right now, as there must not be any divs between the grid layer and the sticker. That seems like an oversight to me, so I asked about context. See https://github.com/ampproject/amphtml/pull/39184#issuecomment-1880878366
Ideally this gets fixed in AMP soon.

In the meantime, if you want to test this locally, you can add an early `return false;` in `Output_Buffer::is_needed()` during development to test the frontend output.

Reply:
Submitted a PR on the AMP repo here: https://github.com/ampproject/amphtml/pull/39727#issuecomment-1881522325

Reply:
This is now resolved and the audio sticker should not be removed anymore